aws/sdk/integration-tests/s3control/Cargo.toml

```diff
@@ -16,4 +16,4 @@ bytes = "1"
 http = "0.2.3"
 serde_json = "1"
 tokio = { version = "1", features = ["full", "test-util"] }
-tracing-subscriber = "0.2.18"
+tracing-subscriber = "0.3.5"
```

Comment:
unless it's been fixed, upgrading to 0.3 requires adding the `env-filter` feature to be equivalent to 0.2